bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -179,3 +179,42 @@ def _setattr(self, name, val):
     @property
     def __version__(self):
         return self.gROOT.GetVersion()
+
+    # Overload VecOps namespace
+    @property
+    def VecOps(self):
+        if not 'VecOps' in self.__dict__:
+            ns = self._fallback_getattr('VecOps')
```

Comment:
@etejedor This is the correct function? Why actually it is called `_fallback_getattr`? However, due to this one we get the same error handling, e.g., if TMVA is not there and you call `ROOT.TMVA.Experimental.whatever`

Reply:
We could use `_fallback_getattr` but that function has more cases than we need. In the end we just want to do a lookup of `VecOps` in cppyy's global namespace, so we could do:
```python
try:
    ns = gbl_namespace.VecOps
except:
    # error out
```

Reply:
But what if the namespace is not there? E.g., you don't have `TMVA` but you call `ROOT.TMVA.something`. We will fall into this mechanism and fail with a different error message than for other namespaces. That was the reasoning for going through the `_getattr_fallback`.

Reply:
So the `fallback` means "fallback for cppyy", I just realized (?) I would go then for the solution which is implemented here.